env/WCOSS2.env

```diff
@@ -13,6 +13,10 @@ step=$1
 export launcher="mpiexec -l"
 export mpmd_opt="--cpu-bind verbose,core cfp"
 
+# Add path to GDASApp libraries
```

Comment:
Maybe add a TODO reflecting that this is only a temporary solution?

Reply:
Thank you @guillaumevernieres for the suggestion.  Done at ce98ffc